src/app/modules/ix-forms/components/ix-input/ix-input.component.ts

```diff
@@ -59,6 +59,13 @@ export class IxInputComponent implements ControlValueAccessor {
     this.value = value;
   }
 
+  formatValue(value: string | number): string | number {
```

Comment:
Instead of this method, maybe we can have getter and setter for `value`. Let me know if you disagree.